xls-workshop-openlane.ipynb

```diff
@@ -162,10 +162,20 @@
         "  n\n",
         "}\n",
         "\n",
+        "\n",
+        "// This is a function to turn the \"adder1\" function into 8-bit input, 8-bit output function\n",
+        "// Refer to \"Bit Slice Expressions\" section in DSLX reference (Bit Slice Expressions: https://google.github.io/xls/dslx_reference/#bit-slice-expressions)\n",
+        "\n",
```

Comment:
I think it might be clearer if we don't put a newline here.

Reply:
Agree. will fix.